geoextent/lib/extent.py

```diff
@@ -10,6 +10,7 @@
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
+logging.disable(logging.INFO)
```

Comment:
I think this use of `logging.disable` is not what it is intended for. From [the function documentation](https://docs.python.org/3.5/library/logging.html#logging.disable):

>  `logging.disable(lvl)` Provides an overriding level lvl for all loggers which takes precedence over the logger’s own level. When the need arises to temporarily throttle logging output down across the whole application, this function can be useful. 

The key words being "temporarily" and "for the whole application". None of which is happening here.

We want to make the logging configurable for users > see https://github.com/o2r-project/geoextent/issues/68

Reply:
Nice idea!

Reply:
I enabled logging. https://github.com/YouQam/geoextent/commit/49e0a709c687bf57e500143574441202b0e28650